pom.xml

```diff
@@ -76,25 +76,19 @@
 		<dependency>
 			<groupId>de.retest</groupId>
 			<artifactId>recheck</artifactId>
-			<version>0.4.1</version>
-		</dependency>
-
-		<dependency>
-			<groupId>de.retest</groupId>
-			<artifactId>retest-utils</artifactId>
-			<version>2.5.0-alpha.2018-09-05</version>
-		</dependency>
-		
-		<dependency>
-			<groupId>de.retest</groupId>
-			<artifactId>retest-core</artifactId>
-			<version>2.5.0-alpha.2018-09-05</version>
+			<version>0.4.3</version>
+			<exclusions>
```

Comment:
If we change the dependency order (first retest, then recheck), we should be able to omit the exclusion, or? [recheck 0.4.3](https://github.com/retest/recheck/blob/v0.4.3/pom.xml) depends on retest-model 2.4.0, whereas [retest 3.0.1](https://bitbucket.org/retest/retest/src/retest-3.0.1/pom.xml) uses version 4.0.4 via [parent-pom 0.3.8](https://bitbucket.org/retest/parent-pom/src/parent-pom-0.3.8/pom.xml).